internal/promotion/runner/builtin/http_requester.go

```diff
@@ -24,6 +24,8 @@ import (
 const (
 	contentTypeHeader = "Content-Type"
```

Comment:
This constant is now only used by tests, I think. Perhaps move it?

Reply:
It is still used in-code on L210.